### PR TITLE
Dockerfile|docker-linaro: FROM v0.1.6 - v1.1.7

### PR DESCRIPTION
**Pull request by pwalkr**
https://github.com/wsbu/toolchain-native/pull/22

### CODE REVIEW

**Comment by DavidZemon** [comment]
The `docker-linaro` script in this repo doesn't use a tag. Can you please add a tag to that script?

**Comment by pwalkr** [comment]
> Please update `docker-linaro` script with a new tag name.

It's already using `:latest` - https://github.com/wsbu/toolchain-linaro/blob/master/docker-linaro
But I can remove the arbitrary `:4.9` tag here: https://github.com/wsbu/toolchain-linaro/blob/5f7c6e3/Dockerfile#L1

**Comment by DavidZemon** [comment]
Yea, sorry - I didn't realize it wasn't use a tag. Can you add a tag? And yes... remove that silly comment please :)

**Comment by pwalkr** [comment]
Care if I take out the whole Dockerfile header comment? It's an example just asking to get out of sync with docker-linaro script.

**Comment by DavidZemon** [comment]
Nope, been thinking about that too. It was dropped in before that script existed.

**Comment by pwalkr** [comment]
Bump - updated version in self-referencing script and removed redundant comment

**Review by DavidZemon — APPROVED** [review]

**Review by mattcaron-redlion — APPROVED** [review]

**Comment by pwalkr** [comment]
One more bump... rebased back to 1.1.6, merge target is now `release-1` since master is off in v2 realm with ubuntu 18.04. Relates to new PR in toolchain-native: https://github.com/wsbu/toolchain-native/pull/23

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-#         -v $HOME/.conan/data:/home/captain/.conan/data \
-#         -v $HOME/.conan/registry.txt:/home/captain/.conan/registry.txt \
-#         -v $HOME/.conan/.conan.db:/home/captain/.conan/.conan.db \
-#         wsbu/toolchain-linaro \
-#         "$@"
-#     ```
-#
-
-FROM wsbu/toolchain-native:v0.1.4
+FROM pfw/toolchain-native-v0.1.6:c72d203
```
Comment (DavidZemon) [inline Dockerfile]: Will approve was this gets updated.

**Review by DavidZemon — CHANGES_REQUESTED** [review]

**Comment by pwalkr** [comment]
Fix'd (removed my debug docker tag name)

**Review by DavidZemon — APPROVED** [review]

**Review by mattcaron-redlion — APPROVED** [review]

**Pull request merged**